File: app/src/main/java/com/danielspeixoto/connect/util/Validate.java
package com.danielspeixoto.connect.util;

import com.danielspeixoto.connect.R;
import com.danielspeixoto.connect.model.pojo.User;

/**
 * Created by danielspeixoto on 2/14/17.
 */

public class Validate {

    public static final String OK = "OK";
    private static String message;

    public static String User(User user) {
        message = OK;
        if (user.getUsername().length() == 0) {
            message = App.getStringResource(R.string.username_must_fill);
        } else if (!user.getUsername().matches("[a-z | 0-9]*")) {
	        message = App.getStringResource(R.string.username_lowercase);
        }
        return message;
    }
}

File: app/src/main/java/com/danielspeixoto/connect/util/Validate.kt
package com.danielspeixoto.connect.util

import com.danielspeixoto.connect.R
import com.danielspeixoto.connect.model.pojo.User
import com.danielspeixoto.connect.model.pojo.Visitor

/**
 * Created by danielspeixoto on 2/14/17.
 */

object Validate {

    val OK = "OK"
    private var message: String = OK

    fun User(user: User): String {
        message = OK
        if (!user.username!!.matches("[a-z | 0-9]*".toRegex())) {
            message = App.getStringResource(R.string.username_lowercase)
        }
        return message
    }

    fun  visitor(visitor: Visitor): String {
        message = OK
        if (visitor.name.isEmpty()) {
            message = App.getStringResource(R.string.name_must_fill)
        }
        return message
    }


}
